### PR TITLE
DM-42163: eo_pipe/cp_verify parity: ptc

### PR DESCRIPTION
**Pull request by plazas**

### CODE REVIEW

File: python/lsst/cp/pipe/ptc/cpExtractPtcTask.py [hunk 1]
```diff
@@ -809,12 +816,15 @@ def measureMeanVarCov(self, im1Area, im2Area, imStatsCtrl, mu1, mu2):
                     Covariance at (dx, dy).
                 nPix : `int`
                     Number of pixel pairs used to evaluate var and cov.
+        rowMeanVariance : `float` or `NaN`
```
Comment (czwa) [inline python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:819]: NaN is just a flavor of float.

File: python/lsst/cp/pipe/ptc/cpExtractPtcTask.py [hunk 2]
```diff
@@ -809,12 +816,15 @@ def measureMeanVarCov(self, im1Area, im2Area, imStatsCtrl, mu1, mu2):
...
+            Variance of the means of each row in the difference image.
+            Taken from `github.com/lsst-camera-dh/eo_pipe`.
 
             If either mu1 or m2 are NaN's, the returned value is NaN.
         """
         if np.isnan(mu1) or np.isnan(mu2):
             self.log.warning("Mean of amp in image 1 or 2 is NaN: %f, %f.", mu1, mu2)
-            return np.nan, np.nan, None
+            return np.nan, np.nan, None, np.nan
```
Comment (czwa) [inline python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:827]: The test above (L587) tests for `is None`, not `np.nan`.  Same point for the other returns below.

**Review by czwa — APPROVED** [review]
Slight None/nan issue, but I think it's a quick fix.

File: python/lsst/cp/pipe/ptc/cpExtractPtcTask.py (hunk 2 above)
Comment (plazas) [reply python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:827]: Changed the test to check for `np.nan`

File: python/lsst/cp/pipe/ptc/cpExtractPtcTask.py [hunk 3]
```diff
@@ -579,8 +584,10 @@ def run(self, inputExp, inputDims, taskMetadata, inputPhotodiodeData=None):
                 # Mask data point at this mean signal level if
                 # the signal, variance, or covariance calculations
                 # from `measureMeanVarCov` resulted in NaNs.
-                if np.isnan(muDiff) or np.isnan(varDiff) or (covAstier is None):
-                    self.log.warning("NaN mean or var, or None cov in amp %s in exposure pair %d, %d of "
+                if (np.isnan(muDiff) or np.isnan(varDiff) or (covAstier is None)
+                   or (rowMeanVariance is np.nan)):
+                    self.log.warning("NaN mean, var or rowmeanVariance, or None cov in amp %s "
```
Comment (czwa) [inline python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:589]: "rowMeanVariance"?  Just for consistency.

**Review by czwa — APPROVED** [review]

**Pull request merged**